pkg/tracker/balance.go

```diff
@@ -29,6 +30,21 @@ func (b *BalanceTracker) String() string {
 	return BalanceTrackerName
 }
 
+func NewBalanceTrackers(logger log.Logger, db db.DataServerProxy, client contracts.ETHClient, accounts []*rpc.Account) []Tracker {
+	balanceTrackers := make([]Tracker, len(accounts))
+	fmt.Fprintf(os.Stdout, "################### len: %v", len(accounts))
```

Comment:
use logger
```suggestion
	logger = log.With(logger, "component", "balance tracker")
	level.Info(logger,....
```

Reply:
Actually, I think this is no longer needed!